recipes/pandas_plink/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pandas_plink" %}
+{% set version = "1.0.12" %}
+{% set sha256 = "4b9f770cbfef9f0f088e3336c74344d5f55856f245009b2f4ea5ad3dc2c49254" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or not py27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cffi >=1.7
+    - libffi
```

Comment:
So I'm not asking about Python. Am only asking about `libffi`. If you need it for the build, it is highly likely you need it in run too. Could you please check?